tdrs-backend/tdpservice/users/api/login.py

```diff
@@ -47,10 +51,10 @@ def decode_payload(self, id_token):
                 key=cert_str,
                 issuer=os.environ["OIDC_OP_ISSUER"],
                 audience=os.environ["CLIENT_ID"],
-                algorithm="RS256",
+                algorithms=["RS256"],
```

Comment:
For posterity, this argument was the breaking change. All other changes are to facilitate testing.